lib/airbrake-ruby/notifier.rb

```diff
@@ -30,7 +30,7 @@ class Notifier
     def initialize(user_config)
       @config = (user_config.is_a?(Config) ? user_config : Config.new(user_config))
 
-      unless [@config.project_id, @config.project_key].all?
```

Comment:
@kyrylo 
This change got me into a bit of a trouble.
I upgraded from `v1.3.1` to `v1.4.1` and it started to complain about `'both :project_id and :project_key are required'`
But I had these variables set in my `.env` file.

After re-checking the changelog I found out, that I had my `project_id` set to `"test"` in development environment.
And now it needs to be an integer.
Maybe you can change the error message to be more explicit?
`':project_id and :project_key are required as integer'`